scripts/release/atomic_pipeline.py

```diff
@@ -0,0 +1,552 @@
+#!/usr/bin/env python3
+
+"""This pipeline script knows about the details of our Docker images
+and where to fetch and calculate parameters. It uses Sonar.py
+to produce the final images."""
+import json
+import os
+import shutil
+from concurrent.futures import ProcessPoolExecutor
+from copy import copy
+from queue import Queue
+from typing import Dict, List, Optional, Tuple, Union
+
+import requests
+import semver
+from opentelemetry import trace
+from packaging.version import Version
+
+from lib.base_logger import logger
+from scripts.evergreen.release.images_signing import (
+    sign_image,
+    verify_signature,
+)
+
+from .build_configuration import BuildConfiguration
+from .build_context import BuildScenario
+from .build_images import process_image
+from .optimized_operator_build import build_operator_image_fast
+
+TRACER = trace.get_tracer("evergreen-agent")
+
+
+def get_tools_distro(tools_version: str) -> Dict[str, str]:
+    new_rhel_tool_version = "100.10.0"
+    default_distro = {"arm": "rhel90-aarch64", "amd": "rhel90-x86_64"}
+    if Version(tools_version) >= Version(new_rhel_tool_version):
+        return {"arm": "rhel93-aarch64", "amd": "rhel93-x86_64"}
+    return default_distro
+
+
+def is_running_in_evg_pipeline():
+    return os.getenv("RUNNING_IN_EVG", "") == "true"
+
+
+def load_release_file() -> Dict:
+    with open("release.json") as release:
+        return json.load(release)
+
+
+@TRACER.start_as_current_span("sonar_build_image")
```

Comment:
can we rename the span names to match the function name?

Reply:
will this not break existing dashboards (if we have them?)